0.5/articles/unit-testing-elements.md

```diff
@@ -308,7 +308,7 @@ We can now run the `wct` command to execute the tests written above. If all goes
 
 <img src="images/unit-testing-elements/wct-tests-passed.png" alt="Tests passes"/>
 
-How about testing if an attribute is the default value we expect it to be? `<core-selector>` supports a multi attribute in case you want to support multiple items being selectable. Let's add this before `done();` along with our other assertions:
```

Comment:
"multi attribute" doesn't scan well...since the reader naturally joins those two words into "multi-attribute".